controllers/admin.js

```diff
@@ -0,0 +1,122 @@
+import models from '../models';
+import hasher from '../helpers/hasher';
```

Comment:
all `auth` related code should be included in a file called `auth.js` change your `hasher.js` to `auth.js`

Reply:
This file is not auth as it only generates a hash and changing it will have to happen after your branch is merged because I presume you the way you used this function is different